scripts/build_emscripten.sh

```diff
@@ -35,6 +35,7 @@ else
 fi
 
 # solbuildpackpusher/solidity-buildpack-deps:emscripten-13
-docker run -v "$(pwd):/root/project" -w /root/project \
+docker run -v "$(pwd):/emsdk/project" -w /emsdk/project \
```

Comment:
While at it, let's also switch to long options. We should prefer them for readability unless there are portability concerns - and with younger tools like docker I don't think there are any.
```suggestion
docker run --volume "$(pwd):/emsdk/project" --workdir /emsdk/project \
```